feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -102,50 +114,66 @@ func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes) *oc.Interface {
 	return i
 }
 
+// configureNetworkInstance creates nonDefaultVRF and dutPort1 to VRF
+func configureNetworkInstance(t *testing.T, dut *ondatra.DUTDevice) {
+	d := &oc.Root{}
+	ni := d.GetOrCreateNetworkInstance(nonDefaultVRF)
+	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	p := dut.Port(t, "port1")
+	i := ni.GetOrCreateInterface(p.Name())
+	i.Interface = ygot.String(p.Name())
+	i.Subinterface = ygot.Uint32(0)
+	gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(nonDefaultVRF).Config(), ni)
+}
+
 // configureDUT configures port1 and port2 on the DUT.
 func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	d := gnmi.OC()
+	for p, dp := range dutPorts {
+		p1 := dut.Port(t, p)
+		i1 := &oc.Interface{Name: ygot.String(p1.Name())}
+		gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), configInterfaceDUT(i1, &dp))
+		if *deviations.ExplicitPortSpeed {
+			fptest.SetPortSpeed(t, p1)
+		}
+	}
```

Comment:
Since this testcase is updated to program NH with MAC, below deviation is required. Can you please add below after L#138? 

                if *deviations.ExplicitIPv6EnableForGRIBI {
			gnmi.Update(t, dut, d.Interface(p1.Name()).Subinterface(0).Ipv6().Enabled().Config(), bool(true))
		}

Reply:
@mohanasm , We don't have any ipv6 traffic or configurations on interfaces for gRIBI , do we still needed this deviation?

Reply:
Yes, it is required. This is already being used in other scripts like TE-2.1, TE-3.7.

Please find references below
https://github.com/openconfig/featureprofiles/blob/main/feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go#L405
https://github.com/openconfig/featureprofiles/blob/f6796e98ad97ac930437dc34e829541094e0b004/feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go#L276

Reply:
updated .